west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
       revision: 12d7647870888e4cb0e421f2b26884c2e76915ac
       path: modules/hal/nordic
     - name: hal_openisa
-      revision: 9bfbe35aad79fdf552dbdae47a894e26be7c022c
+      revision: pull/4/head
```

Comment:
zephyrproject-rtos/hal_openisa#4 has been merged, so please update to the new SHA